docs/farming-&-staking/farming/additional-guides/verify-farmer.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Verified Farmer Discord Role
+sidebar_position: 1
+description: How to verify your Subspace farmer
+keywords:
+    - Farmer
+    - Verify
+    - Farming
+    - GUI
+    - Executable
+---
+
+We are thrilled to announce the **Verified Farmer** role on our [Discord server](https://discord.com/invite/subspace-network)!
+You can track your farmer through the [Astral Block Explorer.](https://explorer.subspace.network/gemini-3h/consensus#/gemini-3h/consensus)
+
+## Eligibility for the Verified Farmer Role
+
+**If you have earned a block or vote reward**, you can now link your Discord account to your farmer wallet through the Astral application to obtain this role. Here’s how to do it:
+
+1. Select **Connect Wallet** in the top right corner of the [Astral Block Explorer.](https://explorer.subspace.network/gemini-3h/consensus#/gemini-3h/consensus)
+![Connect Wallet](/img/doc-imgs/verify-farmer/connect-wallet.png)
+<br/><br/>
+
+2. Sign into your wallet through Subwallet or Polkadot.
+![Select Wallet](/img/doc-imgs/verify-farmer/select-wallet.png)
+
+3. Find the new "Connect Discord" section within the panel.
```

Comment:
Let’s use the name of the section from the screenshot instead of “Connect Discord”.

Reply:
It should be “Get your Farmer role on Discord”